Zend/tests/final_properties/clone/clone_error.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Test that final properties can't be modified after cloning
```

Comment:
I assume cloning in the uninitialized state and assigning then would work though, right?

Reply:
Right! I'll add a test for this case in `clone_success.php`, I was indeed missed out.